tests/assets/test_crawlers/test_picard2_markdup.metrics

```diff
@@ -1,9 +1,112 @@
 ## htsjdk.samtools.metrics.StringHeader
-# picard.sam.markduplicates.MarkDuplicates INPUT=[/lustre/edgeprod/staging/jobs/test_demultiplexing/150723_E00306_0025_BHCHK3CCXX/alignment/10015AT/10015AT0006/LP6002014-DTP_A05_S5_L001.bam] OUTPUT=/lustre/edgeprod/staging/jobs/test_demultiplexing/150723_E00306_0025_BHCHK3CCXX/alignment/10015AT/10015AT0006/LP6002014-DTP_A05_S5_L001_markdup.bam METRICS_FILE=/lustre/edgeprod/staging/jobs/test_demultiplexing/150723_E00306_0025_BHCHK3CCXX/fastq/10015AT/10015AT0006/LP6002014-DTP_A05_S5_L001_markdup.metrics ASSUME_SORTED=true OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VALIDATION_STRINGENCY=LENIENT    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES READ_NAME_REGEX=[a-zA-Z0-9]+:[0-9]:([0-9]+):([0-9]+):([0-9]+).* VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json
+# MarkDuplicates INPUT=[/LP6002014-DTP_A01_S1_L001.bam] OUTPUT=/LP6002014-DTP_A01_S1_L001_markdup.bam METRICS_FILE=/LP6002014-DTP_A01_S1_L001_markdup.metrics ASSUME_SORTED=true OPTICAL_DUPLICATE_PIXEL_DISTANCE=100 VALIDATION_STRINGENCY=LENIENT    MAX_SEQUENCES_FOR_DISK_READ_ENDS_MAP=50000 MAX_FILE_HANDLES_FOR_READ_ENDS_MAP=8000 SORTING_COLLECTION_SIZE_RATIO=0.25 TAG_DUPLICATE_SET_MEMBERS=false REMOVE_SEQUENCING_DUPLICATES=false TAGGING_POLICY=DontTag CLEAR_DT=true DUPLEX_UMI=false ADD_PG_TAG_TO_READS=true REMOVE_DUPLICATES=false DUPLICATE_SCORING_STRATEGY=SUM_OF_BASE_QUALITIES PROGRAM_RECORD_ID=MarkDuplicates PROGRAM_GROUP_NAME=MarkDuplicates READ_NAME_REGEX=<optimized capture of last three ':' separated fields as numeric values> MAX_OPTICAL_DUPLICATE_SET_SIZE=300000 VERBOSITY=INFO QUIET=false COMPRESSION_LEVEL=5 MAX_RECORDS_IN_RAM=500000 CREATE_INDEX=false CREATE_MD5_FILE=false GA4GH_CLIENT_SECRETS=client_secrets.json USE_JDK_DEFLATER=false USE_JDK_INFLATER=false
 ## htsjdk.samtools.metrics.StringHeader
-# Started on: Tue Nov 07 05:56:38 CST 2017
+# Started on: Mon Jan 21 16:47:33 CST 2019
 
 ## METRICS CLASS	picard.sam.DuplicationMetrics
-LIBRARY	UNPAIRED_READS_EXAMINED	READ_PAIRS_EXAMINED	UNMAPPED_READS	UNPAIRED_READ_DUPLICATES	READ_PAIR_DUPLICATES	READ_PAIR_OPTICAL_DUPLICATES	PERCENT_DUPLICATION	ESTIMATED_LIBRARY_SIZE
-	0	0	0	0	0	0	?	
-Unknown Library	4509	536418	5703	799	90990	90990	0.169657	
+LIBRARY	UNPAIRED_READS_EXAMINED	READ_PAIRS_EXAMINED	SECONDARY_OR_SUPPLEMENTARY_RDS	UNMAPPED_READS	UNPAIRED_READ_DUPLICATES	READ_PAIR_DUPLICATES	READ_PAIR_OPTICAL_DUPLICATES	PERCENT_DUPLICATION	ESTIMATED_LIBRARY_SIZE
+Unknown Library	4945	527174	3953	5169	775	79100	57983	0.150077
+
+## HISTOGRAM	java.lang.Double
+BIN	VALUE
```

Comment:
Do we need all of this test data if the parser `break`s when it reaches here?

Reply:
I think there's value in retaining the original format of a test sample input file, leaving it as close to the generated one as possible, but I'll change it if @tcezard and you believe it's better that way.